src/vs/editor/contrib/snippet/snippetVariables.ts

```diff
@@ -182,6 +187,10 @@ export class TimeBasedVariableResolver implements VariableResolver {
 
 	resolve(variable: Variable): string {
 		const { name } = variable;
+		const dayNames = [nls.localize('Sunday', "Sunday"), nls.localize('Monday', "Monday"), nls.localize('Tuesday', "Tuesday"), nls.localize('Wednesday', "Wednesday"), nls.localize('Thursday', "Thursday"), nls.localize('Friday', "Friday"), nls.localize('Saturday', "Saturday")];
```

Comment:
It would be better to keep those as static readonly propoerties because we don't need to recompute those string-arrays all the time.